index.html

```diff
@@ -13,6 +13,14 @@
             <img src="images/Enabled.png" alt="" class="image-icon">
         </nav>
 
+        <nav>
+            <ul class="nav-content">
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+              </ul>
+        </nav>
+
     </header>
 
     <section class="landing-page all-body" id="body-cover">
```

Comment:
- [ ] Kindly consider setting `<div class="ctn">` div in the center of the page as shown in the figma design that in the desktop screen size, it has equal margins on the left and right side. we need to stick to the figma design as much as possible.
your design
<img width="766" alt="Screenshot 2023-07-13 at 10 17 28 PM" src="https://github.com/MarkYegon7/Mark-Portfolio/assets/96683843/40accae7-7f31-430d-8c69-760e9cf9acb5">

figma design
<img width="870" alt="Screenshot 2023-07-13 at 10 17 58 PM" src="https://github.com/MarkYegon7/Mark-Portfolio/assets/96683843/a416ec83-e171-4710-8cd2-dac5f8c72391">

Reply:
Resolved the issue